client/service/src/builder.rs

```diff
@@ -1103,7 +1115,17 @@ ServiceBuilder<
 				.then(|_| Ok(()))));
 			telemetry
 		});
-
+		// prometheus init
+		if let Some(port) = config.prometheus_port {
+			let future = select(
+					sc_prometheus::init_prometheus(port).boxed()
```

Comment:
Bad style, use `sc_prometheus::init_prometheus(port).boxed(),`

Reply:
Why is it bad style? How should I change it?